examples/templates/README.md

```diff
@@ -1,21 +1,45 @@
 ## Templates Guidline
 
-These templates are a base for the new code examples.
+This folder contains templates for new code examples. Every new code example should be made using these templates. 
+
+#### Prerequisite
+To work with code examples (and Handsontable development in general) use at least Node version `15.9.0`. If you have Node Version Manager installed, you can just type in the terminal `nvm use`.
 
 ### Creating a new example
 
-All new code examples should be added to the `/examples/next` directory. Based on them you can create versioned code examples.
+In this guideline, I will walk you through the process of creating new code examples. 
+
+> If you need some inspiration, what can be done with code examples, go ahead see existing examples in `examples/next/docs/` to get inspired!
+
+Firstly, create a new git branch because you will need to commit new examples to the Handsontable repo.
+
+Currently, we are inside the `examples/templates` directory where all templates for future code examples live. You can see that we have templates for examples made in React, Angular, Vue, and of course vanilla Javascript.
+
+To create a code example in vanilla Javascript for our documentation we need to use the template from the `examples/templates/js` directory. Let's make one!
+
+#### 1. Copy ready-made template
+
+Copy the whole `examples/templates/js` directory and paste it into `examples/next/docs/js` directory.
+
+Now rename the copied `js` directory into some meaningful title, like `this-is-some-js-example`. You should have this structure in your codebase:
```

Comment:
the **copied** js directory or **pasted** js directory?

WDYT "In the next step rename the give that 'js' directory to a name that best describes the purpose of the code example, for instance, "sorting-data-custom-order". Make sure you keep the right file structure, like so:"